authors/apps/articles/views.py

```diff
@@ -98,6 +99,16 @@ def update(self, request, *args, **kwargs):
         serializer.update(article, fresh_article_data)
         return Response(serializer.data, status=status.HTTP_200_OK)
 
+    def destroy(self, request, *args, **kwargs):
+        if not Article.objects.filter(pk=kwargs['pk']).exists():
+            raise APIException({"error": "Article Not found"})
```

Comment:
Raise a BadRequest

Reply:
@brucemakallan Why do we still have API Exception here. Make sure that your tests are checking for these status codes.

Reply:
This is now tested for